nginx/default.toml

```diff
@@ -19,6 +19,12 @@ worker_processes = 4
 # worker_connections: Connections per Worker Process.  Default = 1024
 worker_connections = 1024
 
+[[include.additional_configuration_file]]
+config_file = "mime.types"
+
+[[include.additional_configuration_file]]
```

Comment:
Yeah, that's right, I guess I thought there was a better way? I don't use TOML every day, and that looked verbose to me. I'm fine with this as is.